docs/docs/development/setup/zmk.md

```diff
@@ -0,0 +1,128 @@
+---
+title: ZMK Sources
+sidebar_label: ZMK Sources
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const OsTabs = (props) => (<Tabs
+groupId="operating-systems"
+defaultValue="docker"
+values={[
+{label: 'VS Code & Docker', value: 'docker'},
+{label: 'Debian/Ubuntu', value: 'debian'},
+{label: 'macOS', value: 'mac'},
+{label: 'Windows', value: 'win'},
+{label: 'Raspberry OS', value: 'raspberryos'},
+{label: 'Fedora', value: 'fedora'},
+]
+}>{props.children}</Tabs>);
+
+### Source Code
+
+Next, you'll need to clone the ZMK source repository if you haven't already. Navigate to the folder you would like to place your `zmk` directory in and run the following command:
+
+```
+git clone https://github.com/zmkfirmware/zmk.git
+```
+
+:::caution
+If you're using the Docker+VSCode environment on Windows, you _must_ checkout the sources to a folder within `C:\Users\[your_user_here]` to avoid a potential permissions issue.
```

Comment:
This should probably be _above_ the command to check sources out so you see it before you check things out to the wrong place.

If adding the note above about WSL2, also list the WSL2 file system as a valid place to check out source to.